docs/deploy_hostack_on_psi.md

```diff
@@ -0,0 +1,274 @@
+
+# 1. Introduction to Hotstack
+
+- Hotstack is an automation tool engineered to streamline the deployment of lab environments on top of existing OpenStack cloud infrastructure for Red Hat Openstack overcloud (RHOSO) testing and development activities.
+- The Hostack is a collection of Ansible roles and playbooks that manage distinct tasks of deployment.
+- Core Ansible Roles:
+    HotStack's functionality is modularized through several key Ansible roles :
```

Comment:
I think you want to prefix these lines with `-` at the correct indentation level, or with numbers like in the `Key Ansible Playbooks` section?
It currently renders as one long paragraph, not a list.
See: https://github.com/openstack-k8s-operators/hotstack/blob/af8843d7578443bfc130b81d2a135e0d46700713/docs/deploy_hostack_on_psi.md